crates/sui-types/src/nitro_root_certificate.pem

```diff
@@ -0,0 +1,14 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Confirming that downloading from https://docs.aws.amazon.com/enclaves/latest/user/verify-root.html and verifying the sha256 hash matches 8cf60e2b2efca96c6a9e71e851d00c1b6991cc09eadbe64a6a1d1b1eb9faff7c